File: Java/Ram.java
public class Ram{
    
    private String kapasitas; //deklarasi Attribute
    private int harga; //deklarasi Attribute

    // Constructor
    public Ram(){

    }
    public Ram(String kapasitas, int harga) {
        this.kapasitas = kapasitas;
        this.harga = harga;
    }

    // Get dan Set
    public String getKapasitas() {
        return kapasitas;
    }
    public void setKapasitas(String kapasitas) {
        this.kapasitas = kapasitas;
    }
    public int getHarga() {
        return harga;
    }
    public void setHarga(int harga) {
        this.harga = harga;
    }
    
    //keluaran
    public void printRam(){
        System.out.println("---- RAM ----");
        System.out.println("Kapasitas: " + this.kapasitas);
        System.out.println("Harga: " + this.harga);
    }   
}

File: PHP/Ram.php
<?php
class Ram{
    
    private $kapasitas; //deklarasi Attribute
    private $harga; //deklarasi Attribute

    # Constructor 
    function __construct($kapasitas, $harga)
    {
        $this->kapasitas = $kapasitas;
        $this->harga = $harga;
    }
    #set get
    function setKapasitas($kapasitas){
        $this->kapasitas = $kapasitas;
    }
    function getKapasitas(){
        return $this->kapasitas;
    }
    function setHarga($harga){
        $this->harga = $harga;
    }
    function getHarga(){
        return $this->harga;
    }
    
    #keluaran
    function printRam(){
        echo "---- RAM ----". "<br>";
        echo "Capacity: " .$this->kapasitas."<br>";
        echo "Price: " .$this->harga."<br>";
    }
    
    function __destruct()
    {
        
    }
}
?>